### DISCUSSION TITLE
Unable to Submit Form Using Enter Key

### DISCUSSION THREAD

**Discussion by jstarfruits** [orchidsoftware/platform#2825] [Q&A]
I'm facing an issue where pressing the Enter key within a form element doesn't submit the form. It appears that a method called disableKey might be intercepting the Enter key press event. Could someone explain the reason for this behavior and suggest a way to enable form submission using the Enter key? Any assistance on bypassing or disabling the disableKey method would be greatly appreciated. Thank you!

**Comment by anxgstadler** [comment]
Hi!

Did you find an answer to this? I'm in need for a solution to the very same problem :-)
Thanks!

**Reply by jstarfruits** [reply]
@anxgstadler In my environment, the issue was resolved by upgrading to the latest version. If upgrading doesn't improve the situation, it might be due to remaining JavaScript cache.
